src/main/java/org/example/web/MainServlet.java

```diff
@@ -1,38 +1,60 @@
 package org.example.web;
 
+import jakarta.servlet.RequestDispatcher;
+import jakarta.servlet.ServletException;
 import jakarta.servlet.http.HttpServlet;
 import jakarta.servlet.http.HttpServletRequest;
 import jakarta.servlet.http.HttpServletResponse;
-import org.example.constant.RequestArguments;
+import org.example.constant.RequestArgument;
 import org.example.model.Arguments;
+import org.example.model.JspAttribute;
 import org.example.model.Response;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 import java.io.IOException;
+import java.util.List;
 
 public class MainServlet extends HttpServlet {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(MainServlet.class);
     private final PathMapper pageMapService = PathMapper.getInstance();
 
     @Override
-    protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws IOException {
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws IOException, ServletException {
```

Comment:
Remove `throws` and handle exceptions properly.

Reply:
fixed